misk-crypto/src/test/kotlin/misk/crypto/PgpKeyTest.kt

```diff
@@ -19,7 +20,8 @@ import org.assertj.core.api.Assertions.assertThat
 import org.bouncycastle.openpgp.PGPException
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.assertThrows
-import java.util.Base64
+import wisp.deployment.TESTING
+import java.util.*
```

Comment:
nit: star import (is the linter broken)

Reply:
I had "upgraded" the IDE (i.e. complete reinstall) and missed fixing this setting.

Reply:
Fixed